src/libraries/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -3,6 +3,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;netstandard2.0;netcoreapp2.0-Windows_NT</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
+    <!-- Suppress CS0436 to allow ObsoleteAttribute to be internally defined and used -->
+    <NoWarn>$(NoWarn);CS0436</NoWarn>
```

Comment:
Disable this warning for legacy builds only? It would catch the unnecessary local definition of the attribute for NetCoreAppCurrent.